base/src/main/java/io/spine/code/proto/enrichment/BuiltIn.java

```diff
@@ -73,6 +83,16 @@ public boolean test(Descriptor message) {
             return message.getName()
                           .endsWith(typeSuffix);
         }
+
+        @Override
+        public Optional<PackageName> packageName() {
```

Comment:
What if a type reference refers to more than one package, like composite type reference? Or, there are two packages in a short form that match different branches having the common infix?